<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllowKidPlay</class>
 <widget class="QMainWindow" name="AllowKidPlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="childTable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Username</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Password</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Can Play</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="parent_TABLE_LABEL">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;child table&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allowButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Allow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_kid_name">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="kid_NAME_LABEL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter child  Name:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
